scripts/release.sh

```diff
@@ -1,9 +1,35 @@
 #!/usr/bin/env bash
 
-yarn lint &&
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
 
-yarn build &&
+if [ ! -z "${GITHUB_TOKEN}" ]; then
```

Comment:
```suggestion
if [ -z "${GITHUB_TOKEN}" ]; then
```

Double negative!

Reply:
Fixed in https://github.com/cfpb/design-system/pull/2141/commits/5ff5dd63b039e19faf42057d97daabd5e74f85e3